components/property_info.rst

```diff
@@ -303,32 +305,32 @@ Each object will provide 6 attributes, available in the 6 methods:
 ``Type::getBuiltInType()``
```

Comment:
All of these methods are kind of removed:
- `getBuiltinType` is the same as `getTypeIdentifier` of `BuiltinType` and `ObjectType`
- `isNullable` still exists
- `getClassName` only exists on `ObjectType`
- `isCollection` is the equivalent of `$type instanceof CollectionType`
- `getCollection[Key|Value]Types` is the same as `getCollection[Key|Value]Type` of `CollectionType`

The `Type` of `TypeInfo` is really different from the one from `PropertyInfo`, so I think we might rewrite that part completely.

Or maybe redirect to the doc related to TypeInfo's `Type`?